src/main/java/com/mrdb/model/GetAndSetData.java

```diff
@@ -0,0 +1,70 @@
+package com.mrdb.model;
+
+import com.mrdb.entities.UserEntity;
+import com.mrdb.entities.UserEntityRepository;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class GetAndSetData {
```

Comment:
`GetAndSetData` naming is very generic. By reading that name, I cannot know what it does intuitively. 
A better naming for this is `UserService` - this is a service class that performs user related actions (register, login, validations etc). 
In addition, why don't you make this a spring component and autowire the `UserEntityRepository` dependency instead of instantiating it multiple times in the `MainController`?